access/index.html

```diff
@@ -58,17 +58,12 @@
 
 
        <div id="developer">
-                  <p>
-                    <a class="button" href="http://guardian.mashery.com/member/register">Register
-                    </a>
-                  </p>
-
                   <p class="head" style="text-align: center">
-                  By applying to register you are deemed to have accepted the <a class="link" href="http://www.theguardian.com/open-platform/terms-and-conditions">Guardian Open Platform terms and conditions</a><br />
-                  <a class="link" href="https://secure.mashery.com/login/guardian.mashery.com/">Already have a Mashery account?</a>
```

Comment:
```html
 <a class="link" href="https://secure.mashery.com/login/guardian.mashery.com/">Already have a Mashery account?</a>
```
 May stay as it was an alternative link for users already having a mashery account, but I am not sure if it is really useful :wink: